docs/quickstarts/aws.md

```diff
@@ -5,174 +5,226 @@ weight: 10
+export KUBECONFIG=$(pwd)/assets/cluster-assets/auth/kubeconfig
+kubectl get nodes
+```
+
+Sample output:
 
-Apply complete! Resources: 118 added, 0 changed, 0 destroyed.
+```
+NAME                            STATUS   ROLES    AGE   VERSION
+lokomotive-demo-controller-0      Ready    <none>   33m   v1.17.4
+lokomotive-demo-pool-1-worker-0   Ready    <none>   33m   v1.17.4
+lokomotive-demo-pool-1-worker-1   Ready    <none>   33m   v1.17.4
+```
 
-Your configurations are stored in /home/imran/lokoctl-assets/myawscluster
+Verify all pods are ready:
 
-Now checking health and readiness of the cluster nodes ...
+```console
+kubectl get pods -A
+```
 
-Node              Ready    Reason          Message
+Verify you can access httpbin:
 
-ip-10-0-39-75     True     KubeletReady    kubelet is posting ready status
-ip-10-0-39-78     True     KubeletReady    kubelet is posting ready status
-ip-10-0-39-29     True     KubeletReady    kubelet is posting ready status
-ip-10-0-12-241    True     KubeletReady    kubelet is posting ready status
-ip-10-0-12-244    True     KubeletReady    kubelet is posting ready status
-ip-10-0-12-249    True     KubeletReady    kubelet is posting ready status
+```console
+kubectl -n httpbin port-forward svc/httpbin 8080
 
-Success - cluster is healthy and nodes are ready!
+# In a new terminal.
+curl http://localhost:8080/get
 ```
 
-Use the generated `kubeconfig` file to access the Kubernetes cluster and list nodes.
+Sample output:
 
-```console
-export KUBECONFIG=./lokomotive-assets/cluster-assets/auth/kubeconfig
-kubectl get nodes
+```
+{
+  "args": {}, 
+  "headers": {
+    "Accept": "*/*", 
+    "Host": "localhost:8080", 
+    "User-Agent": "curl/7.70.0"
+  }, 
+  "origin": "127.0.0.1", 
+  "url": "http://localhost:8080/get"
+}
 ```
 
 ## Using the cluster
 
-At this point you have access to the Kubernetes cluster and can use it!
-If you don't have Kubernetes experience you can check out the [Kubernetes
-Basics official
-documentation](https://kubernetes.io/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/)
-to learn about its usage.
+At this point you should have access to a Lokomotive cluster and can use it to deploy applications.
+
+If you don't have any Kubernetes experience, you can check out the [Kubernetes
+Basics](https://kubernetes.io/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/) tutorial.
 
-**Note**: Lokomotive sets up a pretty restrictive Pod Security Policy that
-disallows running containers as root by default, check the [Pod Security Policy
-documentation](../concepts/securing-lokomotive-cluster.md#cluster-wide-pod-security-policy)
-for more details.
+>NOTE: Lokomotive uses a relatively restrictive Pod Security Policy by default. This policy
+>disallows running containers as root. Refer to the
+>[Pod Security Policy documentation](../concepts/securing-lokomotive-cluster.md#cluster-wide-pod-security-policy)
+>for more details.
```

Comment:
We can also mention that we deploy this webhook server which disallows usage of default service account.